docs/release-notes/detach-keyboard-config-from-Live.rst

```diff
@@ -0,0 +1,22 @@
+:Type: GUI
+:Summary: Split keyboard configuration between Live media and system to be installed (#2016613)
+
+:Description:
+    The keyboard switching in the Anaconda installer on the Live media did not behave as expected
+    on Wayland based environments. When users changed the keyboard layout configuration that
+    configuration was reflected in the Live environment. However, if users pressed modifier keys
+    (CTRL or SHIFT) the keyboard specified by the Anaconda installer is changed back for the Live
```

Comment:
s/is/was/ to keep the same tense

(The present tense on the next line below is fine, I think, because it describes the long-term state, not step in a process.)

Reply:
Thanks, fixed.